.github/workflows/CI.yml

```diff
@@ -15,11 +15,12 @@ jobs:
     - name: Set up Go
       uses: actions/setup-go@v3
       with:
-        go-version: 1.18
+        go-version: ${{ matrix.go-version }}
```

Comment:
just wondering, there's no matrix strategy definition in this file, where this version comes from?

Reply:
Hmm.. I thought it was used from go.mod, but it seems a bit different. I'll follow the latest version!
https://github.com/terraswap/terraswap-service/pull/26/commits/a480d9e4a53471503f38780b709e1aaaeb1e7be0